src/main/java/com/autotune/analyzer/Analyzer.java

```diff
@@ -59,5 +59,9 @@ public static void addServlets(ServletContextHandler context) {
         context.addServlet(UpdateResults.class, ServerContext.UPDATE_RESULTS);
         context.addServlet(Recommendation.class, ServerContext.RECOMMEND_RESULTS);
         context.addServlet(PerformanceProfileService.class, ServerContext.CREATE_PERF_PROFILE);
-    }
+
+		// Adding UI support API's
+		context.addServlet(ListNamespaces.class, ServerContext.LIST_NAMESPACES);
+		context.addServlet(ListDeploymentsInNamespace.class, ServerContext.LIST_DEPLOYMENTS);
+	}
```

Comment:
Seems this is misaligned

Reply:
I tried to fix it, please lemme know if it's good now.